sdk/common/src/main/java/io/opentelemetry/sdk/resources/ResourceConstants.java

```diff
@@ -25,6 +25,36 @@
  */
 public class ResourceConstants {
 
+  /** The operating system type, such as {@code "WINDOWS"}, {@code "DARWIN"}, {@code "LINUX"}. */
+  public static final String OS_NAME = "os.name";
```

Comment:
Are these attributes defined by semantic convention? If yes, I think a link to it will be useful here.

Reply:
It's in the class javadoc - but thanks to the reminder I found it was a dead link :)